app/service/rest_svc.py

```diff
@@ -249,6 +249,11 @@ def __init__(self):
         with open(file_path, 'r') as f:
             return yaml.load(f.read(), Loader=yaml.FullLoader)
 
+    async def _reload_data_directories(self):
+        plugins = [p for p in await self._services.get('data_svc').locate('plugins', match=dict(enabled=True))]
+        plugins.append(Plugin(data_dir='data'))
```

Comment:
you're adding the 2nd case around treating the core 'data' directory as a temporary plugin. the first is on server boot in server.py. 

we need to make this "automatic" as the more times we repeat this, the more risk we take in forgetting about it.

Reply:
Not sure I'm following. I can move this into the `reload_data` function?